CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,81 @@
+# gittuf Contributor ladder
+
+gittuf's contributor ladder is loosely inspired by the OpenSSF [Scorecard
+Contributor Ladder]. This document details how community members may advance to
+more-involved roles within the gittuf project.
+
+- [Roles](#roles)
+    - [Community members](#community-members)
+    - [Maintainers](#maintainers)
+- [Inactive members](#inactive-members)
+
+## Roles
+
+### Community members
+
+Community members engage with gittuf, contributing their time and energy in
+discussions or just generally helping out.
+
+#### Pre-requisites
+
+- Must follow the [OpenSSF Code of Conduct]
+- Must follow the [Contribution Guide]
+
+#### Responsibilities
+
+- Keep it up!
+
+### Maintainers
+
+Maintainers are responsible for the project's overall health. They are the only
+ones who can approve and merge code contributions.
+
+**Defined by:** Listed in the `MAINTAINERS.txt` file in the project repository
+
+#### Pre-requisites
+
+- Reviewed at least 5 substantial PRs to the codebase
+- Reviewed or got at least 10 PRs merged to the codebase
```

Comment:
Do we want literal limits or should this be more like my suggested text:

```suggestion
- Provided substantial improvements to the codebase (e.g., adding 5 substantial PRs to the codebase)
- Helped to review other submissions to the codebase (e.g., reviewed 10 PRs that were merged to the codebase)
```

Reply:
I also think both of the above lines in your PR should not start with Reviewed.  Do you mean authored, etc.?